pytest_twisted.py

```diff
@@ -51,14 +53,33 @@ def stop_twisted_greenlet():
         gr_twisted.switch()
 
 
-def pytest_addhooks(pluginmanager):
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config):
     global gr_twisted
+    global reactor
+
     if not gr_twisted and not reactor.running:
+        if config.getoption('qt5reactor'):
+            if 'twisted.internet.reactor' in sys.modules:
+                del sys.modules['twisted.internet.reactor']
+
+            import qt5reactor
```

Comment:
Hi guys, `pytest-qt` author here. 😁 

IIUC, `qt5reactor.install()` must be called after `pytest-qt`'s [`qapp` fixture](https://github.com/pytest-dev/pytest-qt/blob/master/pytestqt/plugin.py#L39) correct? Then I think the correct way is to inject a new fixture into the global namespace which depends on `qapp` at this point, something like this:

```python
if config.getoption('qt5reactor'):
    
    class Qt5ReactorPlugin(object):
        
        @pytest.fixture(scope='session', autouse=True)
        def qt5reactor(qapp):
            import qt5reactor
            qt5reactor.install()

    config.pluginmanager.register(Qt5ReactorPlugin())   
```

This is the recommended way of creating new fixtures based on command-line options, and will ensure the `QApplication` object is created before `qt5reactor.install` is called.

But now that I think about it, perhaps `pytest-twisted` authors would also like to create a generic `reactor` fixture which is selected from a `--reactor` command-line option? Full disclosure, I've never used twisted and have only a vague idea of what a "reactor" is, so excuse me if that last thought didn't make much sense.